ui/src/css/storyTelling.css

```diff
@@ -48,7 +48,21 @@ fieldset ul {
 
 .sjrk-st-page-content-container {
     background-color: #fff;
-    padding: 2rem;
+    padding: 0 2rem 2rem 2rem;
+}
+
+/* These sections are shown or hidden based on the theme's styling rules */
+.sjrk-st-page-header-container,
```

Comment:
Is this rule to hide these containers created because these containers are not expected to show in Karisma theme? Can you add a comment to explain? Thanks.